upgrade/README.md

```diff
@@ -1,23 +1,54 @@
 # How to run the upgrade test
 
-- Create a cluster and set it as the current cluster.
+## Create a test cluster
+
+Create a cluster and set it as the current cluster.
 You may use `kubectl config current-context` to confirm that the cluster
 is as expected.
 
-- Configure the upgrade test.
+## Configure the test options
+
 Let the root directory of this repo be *ROOT-OF-REPO*.
-Run the following commands, in which *SOURCE_TAG* specifies the
-version of Istio to be upgraded and *TARGET_TAG* specifies the
-version of Istio to which the test will upgrade.
-Note that you may need to configure the variables in the command based
-on your test cases.
+Enter the directory containing the tests.
 
     ```bash
     cd ROOT-OF-REPO/upgrade
-    export SOURCE_TAG=1.5_latest; export TARGET_TAG=master; export INSTALL_OPTIONS=istioctl; export UPGRADE_TEST_LOCAL=true;
     ```
 
-- Run the upgrade test.
+You may choose three types of test scenarios by configuring
+the TEST_SCENARIO variable. In the following example commands,
+*SOURCE_TAG* specifies the
+version of Istio to be upgraded/downgraded and *TARGET_TAG* specifies the
+version of Istio to which the test will upgrade/downgrade.
+Note that you may need to configure the variables in the commands based
+on your test cases.
+
+* When TEST_SCENARIO is configured as *upgrade-downgrade*,
+during testing, Istio will be upgraded and then downgraded.
```

Comment:
nit: `during testing` can be removed for concise. same for others

Reply:
Done.